multiple-erc20-to-erc20/contracts/contracts/upgradeable_contracts/factories/BasicBridgeFactory.sol

```diff
@@ -4,6 +4,7 @@ import "../../upgradeability/EternalStorage.sol";
 import "../EternalOwnable.sol";
 
 contract BasicBridgeFactory is EternalStorage, EternalOwnable {
+    uint256 constant public defaultDecimals = 18;
```

Comment:
@LiorRabin It's ok to put it here (regarding upgradability)?

Reply:
I think it's fine